Makefile.authx

```diff
@@ -62,12 +62,25 @@ clean-tyk:
 	-docker volume rm tyk-redis-data
 
 
+#>>>
+# close federation service
+# make clean-federation
+
+#<<<
+clean-federation:
+	source setup_hosts.sh; \
+	docker compose -f lib/candigv2/docker-compose.yml -f lib/federation/docker-compose.yml down || true
+
+	# - remove intermittent docker images
```

Comment:
I know that this is in all of the other clean-% targets, but I think we should not be removing docker images here. They get cleaned up in clean-all (as part of clean-images).

Reply:
I wrote this suggestion into https://candig.atlassian.net/browse/DIG-1171, so maybe we will eventually do this for all of the authx targets.